pennylane/2_Graph_optimization_with_QAOA.ipynb

```diff
@@ -593,7 +593,7 @@
     "\n",
     "<div class=\"alert alert-block alert-warning\">\n",
     "<b>Caution:</b> Increasing the maximum number of parallel executions can result in a greater rate of spending on simulation fees. The value must also be set bearing in mind your\n",
-    "    service <a href=\"https://docs.aws.amazon.com/braket/latest/developerguide/braket-quotas.html\">quota</a>, which can be found <a href=\"https://console.aws.amazon.com/servicequotas/home\">here</a>.\n",
+    "    service <a href=\"https://docs.aws.amazon.com/braket/latest/developerguide/braket-quotas.html\">quota</a>, which can be found <a href=\"https://console.aws.amazon.com/servicequotas/home\">here</a>. Besides, if you choose to run the QAOA problem on a QPU, it can take a long time and incur high <a href=\"https://aws.amazon.com/braket/pricing/\">usage fees</a>. \n",
```

Comment:
```suggestion
    "    service <a href=\"https://docs.aws.amazon.com/braket/latest/developerguide/braket-quotas.html\">quota</a>, which can be found <a href=\"https://console.aws.amazon.com/servicequotas/home\">here</a>. If you instead choose to run the QAOA problem on a QPU, it can take a long time and incur high <a href=\"https://aws.amazon.com/braket/pricing/\">usage fees</a>. \n",
```